simulations/llm_ig_simulation/src/benchmark_one_server.py

```diff
@@ -0,0 +1,258 @@
+
+        
+        
+        
```

Comment:
Remove extra lines (here and other places)

Reply:
done